src/aap_eda/services/activation/status_manager.py

```diff
@@ -62,7 +62,7 @@ class StatusManager:
 
     def __init__(
         self,
-        db_instance: tp.Union[models.Activation, models.EventStream],
+        db_instance: models.Activation,
```

Comment:
The `EVENT_STREAM` had been removed from the `ProcessParentType` enum, we should remove the check on https://github.com/ansible/eda-server/pull/1008/files#diff-0140165e8f538211fec9cbc973dca39efb2e8c248cff89b4867cfa7c174d189cR73